sig-ui/annual-report-2021.md

```diff
@@ -0,0 +1,102 @@
+# SIG UI 2021 Annual Report
+
+**Authors:**
+
+Sebastian Florek ([@floreks](https://github.com/floreks))
+Marcin Maciaszczyk ([@maciaszczykm](https://github.com/maciaszczykm))
+Jeffrey Sica ([@jeefy](https://github.com/jeefy))
+
+- [Operational](#operational)
+- [Membership](#membership)
+- [Current initiatives and project health](#current-initiatives-and-project-health)
+
+## Operational
+
+**How are you doing with operational tasks in [sig-governance.md](/committee-steering/governance/sig-governance.md)?**
+
+The SIG primarily works asynchronously. In the past, our meetings would last less than five minutes and be relatively disruptive. Since the release of the 2.0 Dashboard, meeting attendance transitioned from optional to near-non-existent. Instead, we coordinate through the #sig-ui Slack channel and handle any project planning via GitHub. All chairs are regular contributors of the Dashboard subproject and help review PRs and triage issues. 
+
+**Is your README accurate? have a CONTRIBUTING.md file?**
+
+Yes, our README is accurate and is typically updated with every new Dashboard release. Also we do have a CONTRIBUTING.md file however it has not been updated with any more info beyond the standard template.
+
+**All subprojects correctly mapped and listed in [sigs.yaml](/sig-list.md)?**
+
+Yes. The only subproject under SIG-UI is the Kubernetes Dashboard. There is an additional companion service that gets deployed to feed metrics into the Dashboard (Dashboard Metrics Scraper) but it isn't considered a subproject by the SIG.
```

Comment:
It's mentioned in https://github.com/kubernetes/community/tree/master/sig-ui#dashboard... so should this be updated to state that the metrics scraper is also considered part of the `dashboard` subproject?

Reply:
https://github.com/kubernetes-sigs/dashboard-metrics-scraper appears to be WIP and slightly inactive. Can you add details around what's the plan/roadmap for this repo?

Reply:
We could remove the WIP mark. Overall there is no roadmap for the metrics scraper. It does its' job and we don't plan to extend it. It can be used by anyone and is not tightly coupled with the k8s dashboard. I wouldn't consider it a subproject for us. Similarly, we also mention metrics-server as a general metrics provider.

Reply:
I think its okay as part of the dashboard subproject as they are almost always deployed alongside each other. IIRC it was intended as a stopgap solution because the metrics server doesn't save metrics. It's long-term plan or a plan for metrics might be good to think about though.

Reply:
> IIRC it was intended as a stopgap solution because the metrics server doesn't save metrics.

True.

Reply:
can we document on the repo README or somewhere in the SIG README because that's a roadmap: "It does its' job and we don't plan to extend it."